src/Paragraph/Paragraph.ts

```diff
@@ -0,0 +1,47 @@
+import { css, Theme } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import Text from '../Text';
+import { FontSize } from '../theme/typography';
+
+const VARIANT_MAPPING = {
+  ui: 'size4',
+  longForm: 'size5',
+  tiny: 'size3'
+} as const;
+
+type Variant = 'ui' | 'longForm' | 'tiny';
+
+const variantStyles = ({
+  theme,
+  variant,
+  fontSize
+}: {
+  theme: Theme;
+  variant: Variant;
+  fontSize?: FontSize;
+}) => {
+  const fontSizeOverride =
+    fontSize && theme.fontSizes[fontSize]
```

Comment:
Since this shows as a new file rather than a rename, calling out that I've added the `fontSize` presence check (`fontSize && `) to satisfy the compiler.